content/changelog/2024-02-23-console.md

```diff
@@ -0,0 +1,109 @@
+
+<video autoPlay playsInline muted loop width="800" height="600">
+  <source type="video/mp4" src="/docs/relnotes/navigation_breadcrumb.mp4"/>
+</video>
+
+Some other nice updates include:
+
+- Added bold page titles to more easily identify which page you are on as you move through the Neon Console. If you click through to the **Operations** page or **Integrations** page for example, through whatever path you take to get there, you'll better know where you are.
+- We've moved your profile avatar to the header of the page, giving you easier access to your account settings and API keys.
+
+### Neon Vercel Integration improvements
+
+The following changes were made to the [Neon Vercel Integration](https://vercel.com/integrations/neon):
+
+- To ensure that users accessing a Neon database from a serverless environment have enough connections, the `DATABASE_URL` and `PGHOST` environment variables added to a Vercel project by the Neon integration are now set to a pooled Neon connection string by default. Pooled connections support up to 10,000 simultaneous connections. Previously, these variables were set to an unpooled connection string supporting fewer concurrent connections.
+  
+  For tools or applications that require a direct database connection, we added two new environment variables that are set to an unpooled connection string: `DATABASE_URL_UNPOOLED` and `PGHOST_UNPOOLED`.  If necessary, you can configure the Neon Vercel Integration to use these variables. See [Manage Vercel environment variables](/docs/guides/vercel#manage-vercel-environment-variables) for more information. 
+
+  For information about "pooled" vs "unpooled" connections in Neon, see [Connection pooling](https://neon.tech/docs/connect/connection-pooling).
+- Adjustments were made to ensure that the integration correctly branches from the primary branch of your Neon project after changing the primary branch.
+- Resolved a problem encountered during the Neon Vercel Integration setup that caused inconsistencies in the branch, roles, and databases displayed when selecting a Neon project.
+- Improved reliability in keeping environment variables set by the Neon Vercel Integration synchronized with those set in your Vercel project.
+
+To learn more about the Neon Vercel Integration, see [Connect with the Neon Vercel Integration](https://neon.tech/docs/guides/vercel).
+
+### Prisma: One connection to rule them all
+
+Connecting to Neon is simplified with Prisma ORM 5.10.0 or higher. You can now use a **single pooled connection string** when connecting from Prisma to your Neon database. Previously, to use connection pooling with Neon, you had to configure two connections: a pooled database connection for Prisma Client and a direct database connection for Prisma Migrate (using Prisma’s `directUrl` datasource field). Additionally, with Prisma ORM 5.10.0 or higher, you no longer have to append a `pgbouncer=true` option to a pooled Neon connection string.
```

Comment:
`pgbouncer=true` for pooled URL is not required anymore for all Prisma versions. It was a dependency to pgbouncer 1.22.0